data/35e/wizards_of_the_coast/supplement/savage_species/ss_classes.lst

```diff
@@ -105,18 +107,74 @@ CLASS:Djinni Monster Class	STARTSKILLPTS:8	CSKILL:Appraise|Concentration|TYPE=Cr
 8	BONUS:STAT|WIS|2
 #8	BONUS:VAR|DjinniAcidResist|5
 8	BONUS:VAR|AcidResistanceBonus|5
-8	SAB:TODO Whirlwind 3/day
+#8	SAB:TODO Whirlwind 3/day
 #
 # gain 8+int mod skill points
 # when the Djinni becomes large size, it gets -4 to hide checks
 9	BONUS:COMBAT|BASEAB|1|TYPE=BASE.REPLACE
 9	BONUS:STAT|STR|2
+#9	BONUS:STAT|STR|-14|TYPE=Size
 9	SPELLS:Innate|TIMES=1|CASTERLEVEL=13+TL-CLASSLEVEL=Djinni Monster Class|Major Creation|Persistent Image|Wind Walk
-9	SAB:-4 hide check due to size
+#9	SAB:-4 hide check due to size
 9	BONUS:COMBAT|REACH|5
 # cant use SIZE in a class file
 #9	SIZE:L
 9	BONUS:SIZEMOD|NUMBER|1
+# prevent this size change affecting all the stats
```

Comment:
These comments add unnecessary clutter from another file. Please remove and keep the code clean.

Reply:
removed